sdk/src/beta9/runner/endpoint.py

```diff
@@ -6,6 +6,7 @@
 from http import HTTPStatus
 from typing import Any, Dict, Optional, Tuple
 
+import nest_asyncio
```

Comment:
see above comment about removing the dep. Also, we should test this fix with functions/taskqueues and see what happens